Content.Server/Mind/Components/MindComponent.cs

```diff
@@ -137,18 +137,25 @@ public void Examine(FormattedMessage.Builder message, bool inDetailsRange)
                 _entMan.TryGetComponent<MobStateComponent?>(Owner, out var state) &&
                 state.IsDead();
 
-            if (!HasMind)
+            if (dead)
+            {
+                if (Mind?.Session == null) {
+                    // Player has no session attached and dead
+                    message.AddMarkup($"[color=yellow]{Loc.GetString("mind-component-no-mind-and-dead-text", ("ent", Owner))}[/color]");
+                } else {
+                    // Player is dead with session
+                    message.AddMarkup($"[color=red]{Loc.GetString("comp-mind-examined-dead", ("ent", Owner))}[/color]");
+                }
+            }
+            else if (!HasMind)
             {
                 var aliveText =
                     $"[color=purple]{Loc.GetString("comp-mind-examined-catatonic", ("ent", Owner))}[/color]";
-                var deadText = $"[color=red]{Loc.GetString("comp-mind-examined-dead", ("ent", Owner))}[/color]";
 
-                message.AddMarkup(dead ? deadText : aliveText);
+                message.AddMarkup(aliveText);
```

Comment:
One last thing that'd be nice would be to remove the aliveText variable and just put it in the AddMarkup.

Also same for below.

I'll merge it either way in a few hours when I have time to test the PR.

Reply:
Made the change, will take more liberty in condensing in the future